src/components/toast/toast.style.js

```diff
@@ -39,7 +40,13 @@ const ToastStyle = styled(MessageStyle)`
       transition: all 150ms ease-out;
     }
   `}
-  
+
+  ${StyledIconButton} {
```

Comment:
Since `position: absolute` has been removed from base `StyledIconButton` styling. Should't the `absolute` be here?

Reply:
fixed e554b8b